orttraining/orttraining/core/graph/gradient_schema_defs.cc

```diff
@@ -1300,30 +1326,26 @@ Return true if all elements are true and false otherwise.
       .SinceVersion(1)
       .SetSupportLevel(OpSchema::SupportType::EXPERIMENTAL)
       .SetDoc("Send data tensor to the specified destination.")
-      .Input(0, "InputSignal", "Input control signal.", "TBool")
-      .Input(1, "Data", "Tensor to send.", "T")
+      .Input(0, "Remote", "Remote dst rank.", "TInt64")
```

Comment:
>"Remote", "Remote dst rank.", "TInt64") [](start = 16, length = 39)

Do we really have the need to use one send op to communicate with multiple remotes?

Reply:
Only one remote.